README.md

```diff
@@ -0,0 +1,54 @@
+# conference
+Project: HTML/CSS & JavaScript capstone project - Conference page.
+ 
+This Project is based on an online website for a conference. 
+A responsive design
+ 
+![screenshot](https://github.com/aliabbani/Conference-page/blob/mobile-first/images/readme.png)
+ 
+ 
+## Built With
+- HTML
+- CSS
+- JavaScript
+- Coffee
+ 
+## Live Demo
+[Live Demo Link](https://aliabbani.github.io/Conference-page/)
+
+
+## Getting Started
+
+- Clone the repo to your local machine.
+- Open the index.html file on your browser and use.
+
+
+## Authors
+
+- Name: Ali Abbani
+- GitHub: [@githubhandle](https://github.com/aliabbani)
+- Twitter: [@twitterhandle](https://twitter.com/aliabbani)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/ali-abbani-8b6246150/)
+ 
+            
+## 🤝 Contributing
+ 
+Contributions, issues, and feature requests are welcome!
+ 
+Feel free to check the [issues page](issues/).
+ 
+## Show your support
+ 
+Give a ⭐️ if you like this project!
+ 
+## Acknowledgments
```

Comment:
- [ ] OPTIONAL
It's a nice and smart thing to give appropriate credit to C**indy Shin**, the author of the original design, as required by the Creative Commons License. Kindly ensure to add this under the "Acknowledgements" section of your ReadMe

Reply:
Alright, thank you!